test/unix/dune

```diff
@@ -1,3 +1,3 @@
 (tests
  (names main force_merge)
- (libraries index.unix alcotest fmt logs logs.fmt))
+ (libraries index.unix alcotest fmt logs logs.fmt str))
```

Comment:
I think the `Str` module is a bit unpopular (https://github.com/mirage/mirage/issues/663). Since it's only a test dependency, it maybe doesn't matter that much, but I think it's generally to be avoided.

Reply:
Thanks, I didn't know. I'll take a look at Astring.

Reply:
it's unpopular because it is not thread safe (and the API is unsafe) ;-)

Reply:
for regular expression matching, it's probably better to use `Re`